ibc/simapp/ante_handler.go

```diff
@@ -0,0 +1,55 @@
+package simapp
```

Comment:
Could we just use the simapp folder from cosmos sdk or is there something special about this one?

Reply:
There are a small amount of helpers that are needed from the simapp directory, but not the app itself, see https://github.com/cosmos/interchain-security/pull/636

Reply:
LGTM